Yoast/wordpress-seo
# Add Spotify to social profile URLs

## Summary

This PR can be summarized in the following changelog entry:

* Add Spotify as option in social profile URLs.

## Relevant technical choices:

* Note that while Spotify is included in the social profiles in the schema data, Google does not currently support Spotify in the Knowledge Graph (https://developers.google.com/search/docs/data-types/social-profile). 

## Test instructions

This PR can be tested by following these steps:

To test for a Person:
* Install and activate Yoast SEO. 
* Go to SEO > Search Appearance.
* Select "Person" under Knowledge Graph and click "Save Changes".
* Go to Users > Your Profile.
* Add a Spotify profile URL to "Spotify Profile URL" and click "Update Profile". 
* View the site.
* Open Web Inspector.
* Locate the JSON script in the <head> with the Yoast schema graph data.
* See the Spotify Profile URL listed in the SameAs array.

To test for an Organization:
* Install and activate Yoast SEO. 
* Go to SEO > Search Appearance.
* Select "Organization" under Knowledge Graph.
* Enter an Organization Name and upload an Organization Logo. 
* Click "Save Changes".
* Go to SEO > Social.
* Add a Spotify URL to "Spotify URL" and click "Update Profile". 
* View the site.
* Open Web Inspector.
* Locate the JSON script in the <head> with the Yoast schema graph data.
* See the Spotify URL listed in the SameAs array.

## UI changes

## Documentation

## Quality assurance

I have modified the existing unit test for social profile URLs.

Fixes #4482
